### PR TITLE
Add simple react app

### PR DESCRIPTION
**Pull request by vaddya**

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -1 +1,2 @@
 include("api")
+include("web")
```
Comment (lamtev) [inline settings.gradle.kts]: `include` function takes vararg as an argument, so It's possible to enumerate all modules In one `include`. I don't know which approach is better, but I usually use single `include`

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# accula
\ No newline at end of file
+# Advanced Code Clones UniversaL Analyzer
```
Comment (lamtev) [inline README.md]: Not issue-related change

Comment (vaddya) [reply README.md]: I think it doesn't matter in this case if you aren't going to create a separate task to update README

File: settings.gradle.kts (hunk 1 above)
Comment (vaddya) [reply settings.gradle.kts]: Thanks!

File: web/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```
Comment (lamtev) [inline web/tsconfig.json]: Why not an es6 or even newer one?

File: web/.prettierrc [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "es5",
```
Comment (lamtev) [inline web/.prettierrc]: es5?

Comment (vaddya) [reply web/.prettierrc]: https://prettier.io/docs/en/options.html#trailing-commas

File: web/.gitignore [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.idea
```
Comment (lamtev) [inline web/.gitignore]: this pattern is already covered by root gitignore

File: web/.gitignore [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+/yarn
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
```
Comment (lamtev) [inline web/.gitignore]: this pattern is already covered by root gitignore

File: web/tsconfig.json (hunk 3 above)
Comment (vaddya) [reply web/tsconfig.json]: For instance, if you want to target IE11, you'll have to target ES5. If you want to support NodeJS, Edge or Chromium, ES6 is ok.
I think it's good practice to support older versions of a browser, so let's stick with es5 for now.

File: web/.gitignore (hunk 5 above)
Comment (vaddya) [reply web/.gitignore]: Yeah, it was missing when I added it. Fixed

File: web/.gitignore (hunk 6 above)
Comment (vaddya) [reply web/.gitignore]: Yeah, it was missing when I added it. Fixed

File: web/.eslintignore [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# don't ever lint node_modules
+node_modules
+# don't lint build output (make sure it's set to your correct build folder name)
+dist
+public
+# don't lint nyc coverage output
+coverage
```
Comment (lamtev) [inline web/.eslintignore]: Let's not to break traditions and append new line here

Comment (vaddya) [reply web/.eslintignore]: Done

File: web/package.json [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
```
Comment (lamtev) [inline web/package.json]: MS edge?

File: web/src/index.scss [hunk 9]
```diff
@@ -0,0 +1,5 @@
+body {
+  h1 {
+    
+  }
+}
```
Comment (lamtev) [inline web/src/index.scss]: new line

File: web/package.json (hunk 8 above)
Comment (vaddya) [reply web/package.json]: No idea, auto-generated

File: web/tsconfig.json (hunk 3 above)
Comment (lamtev) [reply web/tsconfig.json]: This was my misunderstanding. Agree with you

File: web/src/index.scss (hunk 9 above)
Comment (vaddya) [reply web/src/index.scss]: Done

File: web/package.json (hunk 8 above)
Comment (vaddya) [reply web/package.json]: Changed to
```
  "browserslist": [
    "defaults"
  ]
```

**Pull request merged**